packages/camera/camera/lib/src/camera_image.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-part of 'camera.dart';
+part of camera;
```

Comment:
Should this file live in the platform_interface instead?

Reply:
As the `CameraImage` class is only used by the `startImageStream` and `stopImageStream` methods which will not be available in the camera_platform_interface this class also doesn't have to be moved.